agents/node/vcxagent-core/readme.md

```diff
@@ -0,0 +1,15 @@
+# VCX Agent Core
+VCX Agent Core is implementation of simple agent with persistent file storage. It's based on
+[NodeJS Wrapper](../../../wrappers/node) for `aries-vcx` library.
+
+# Try it
+1. First step is to [compile](../../../libvcx) `aries-vcx` library and make it available on your system.
+2. You need to have `aries-vcx` compatible agency ready. See more [info](../../../README.md).
```

Comment:
"`aries-vcx`-compatible" or even better "agency compatible with `aries-vcx`" (or aries-compatible, although that's not strictly true)